runtime/drivers/models.go

```diff
@@ -32,6 +32,8 @@ type ModelExecutorOptions struct {
 	InputHandle      Handle
 	InputConnector   string
 	InputProperties  map[string]any
+	StageConnector   string
+	StageProperties  map[string]any
```

Comment:
Can we omit these here? Since the use of the staging connector is contained in the reconciler, and the individual model executor should only need to deal with moving data from the provided input connector to the output connector. 

Maybe it can't be avoided eventually, but it would be nice to remove it to keep model executor implementation simpler and not give an indication of too many special cases that needs handling.